jni/tests/faiss_stream_support_test.cpp

```diff
@@ -7,7 +7,7 @@
 // Modifications Copyright OpenSearch Contributors. See
 // GitHub history for details.
 
-#include "faiss_stream_support.h"
```

Comment:
Should we have cpp test for nmslib stream support as well?

Reply:
sure, unit test for nmslib will be included in the next commit!